draft-ietf-rats-corim.md

```diff
@@ -1408,6 +1418,10 @@ The following describes each member of the `concise-bom-tag` map.
 * `bom-validity` (index 2): Specifies the validity period of the CoBOM.
   Described in {{sec-common-validity}}.
 
+* `profile` (index 3): An optional profile identifier that overrides the
```

Comment:
Since CoBOMs contain pointers to other tags each potentially sporting its own profile, is there a "homogeneity" requirement?  I.e., that the pointed tags have the same profile?

Reply:
No, the scope is to the lexical structure itself. It does not have "dynamic scope" semantics. Added a line.